.bingo/Variables.mk

```diff
@@ -29,11 +29,11 @@ $(CONTROLLER_GEN): $(BINGO_DIR)/controller-gen.mod
 	@echo "(re)installing $(GOBIN)/controller-gen-v0.14.0"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=controller-gen.mod -o=$(GOBIN)/controller-gen-v0.14.0 "sigs.k8s.io/controller-tools/cmd/controller-gen"
 
-GINKGO := $(GOBIN)/ginkgo-v2.15.0
+GINKGO := $(GOBIN)/ginkgo-v2.17.1
```

Comment:
non-blocking, but would it be worth it to put the ginkgo version in a variable?

Reply:
@perdasilva This is an auto generated file by the `bingo` tool